recipes/libmnl/all/conanfile.py

```diff
@@ -0,0 +1,66 @@
+import os, glob
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+class LibmnlConan(ConanFile):
+    name = "libmnl"
+    license = "LGPL-2.1-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://netfilter.org/projects/libmnl/index.html"
+    description = "Minimalistic Netlink communication library"
+    topics = ("libmnl", "netlink")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def configure(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("libmnl is only supported on Linux")
```

Comment:
ftp://ftp7.freebsd.org/sites/www.linuxfromscratch.org/blfs/view/systemd/basicnet/libmnl.html

```suggestion
        if not self.settings.os in ["Linux", "FreeBSD"]:
            raise ConanInvalidConfiguration("libmnl is not supported on {}".format(self.settings.os))
```

Reply:
Addressed rest of your feedback. Their git repo says it's linux only. https://git.netfilter.org/libmnl/tree/configure.ac#n21

Reply:
Gross! Okay that's fine.